packages/components/src/ProgressBar/Sizes.css

```
@@ -0,0 +1,7 @@
+.smaller {
```

Do we anticipating having a `small` size in the future that would be between `base` and `smaller`? If not this might be better as just `small` since this is what a user of the component will need to specify when they are using it and specifying a size.

cc. @chris-at-jobber 

```suggestion
.small {
```